### PR TITLE
edit for online getting_started

### PR DESCRIPTION
**Pull request by sallyom**
WIP @ahardin

### CODE REVIEW

File: getting_started/beyond_the_basics.adoc [hunk 1]
```diff
@@ -376,20 +376,16 @@ endif::[]
 == What's Next?
 
 The following sections provide some next steps now that you have finished your
-initial walkthrough of {product-title} (Next Gen) Developer Preview.
+initial walkthrough of {product-title} {product-version}.
 
 [[btb-developer-preview-considerations]]
 === Developer Preview Usage Considerations
 
-Please review the xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[Developer Preview
```
Comment (sallyom) [inline getting_started/beyond_the_basics.adoc]: @ahardin not sure if the goal is to edit 'devpreview' refs out?

File: getting_started/developers_cli.adoc [hunk 2]
```diff
@@ -140,13 +140,7 @@ ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
 not have access, contact your cluster administrator.
 endif::[]
 ifdef::openshift-online[]
-- You must register for an account with the https://openshift.com[{product-title} (Next Gen) Developer Preview.]
-- Your project must be populated with the
```
Comment (sallyom) [inline getting_started/developers_cli.adoc]: @ahardin I don't think Ls 144-149 are necessary?

File: getting_started/developers_cli.adoc [hunk 3]
```diff
@@ -140,13 +140,7 @@ ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
...
-xref:../dev_guide/templates.adoc#using-the-instantapp-templates[Instant App
-templates] and xref:../using_images/s2i_images/index.adoc#using-images-s2i-images-index[builder images] that
-conform to the current xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[set of
-considerations] for running in the {product-title} (Next Gen) Developer Preview
-offering.
+- You must register for an account with the https://openshift.com[{product-title} {product-version}.]
```
Comment (sallyom) [inline getting_started/developers_cli.adoc]: @ahardin have to check if this is correct link to register for an online account?

File: getting_started/developers_cli.adoc [hunk 4]
```diff
@@ -161,8 +155,6 @@ endif::[]
 ifdef::openshift-dedicated[]
 contact your cluster administrator.
 endif::[]
-- You must have an account on http://www.github.com[GitHub], have https://help.github.com/articles/set-up-git/[Git set up], and
```
Comment (sallyom) [inline getting_started/developers_cli.adoc]: github account not necessary, right?

File: getting_started/beyond_the_basics.adoc (hunk 1 above)
Comment (ahardin-rh) [inline getting_started/beyond_the_basics.adoc]: @sallyom Thanks for these changes! Yes, that is part of the review process -- identifying outdated and soon-to-be outdated references and removing them.

File: getting_started/developers_cli.adoc (hunk 3 above)
Comment (ahardin-rh) [inline getting_started/developers_cli.adoc]: @sallyom This looks correct :star:

**Comment by ahardin-rh** [comment]
@sallyom This looks good to me. Thanks for your help! Is this ready to merge?

**Comment by sallyom** [comment]
@ahardin-rh thanks for your review, it should be ready for merge now, thanks!

File: getting_started/developers_cli.adoc (hunk 3 above)
Comment (tiwillia) [inline getting_started/developers_cli.adoc]: Is it? https://openshift.com/online/starter ? Is that what this maps to?

I think the proper URL would be https://manage.openshift.com

File: getting_started/developers_cli.adoc [hunk 5]
```diff
@@ -140,13 +140,7 @@ ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
...
 endif::[]
 ifdef::openshift-online[]
-- You must register for an account with the https://openshift.com[{product-title} (Next Gen) Developer Preview.]
-- Your project must be populated with the
-xref:../dev_guide/templates.adoc#using-the-instantapp-templates[Instant App
-templates] and xref:../using_images/s2i_images/index.adoc#using-images-s2i-images-index[builder images] that
-conform to the current xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[set of
-considerations] for running in the {product-title} (Next Gen) Developer Preview
-offering.
+- You must register for an account with the https://manage.openshift.com.]
```
Comment (sallyom) [inline getting_started/developers_cli.adoc]: @tiwillia @ahardin-rh got it, thanks

Comment (tiwillia) [inline getting_started/developers_cli.adoc]: Should probably be `You must register for an account with https://manage.openshift.com`

The period on the end will break the URL, and the `the` isn't required.

File: getting_started/developers_cli.adoc [hunk 6]
```diff
@@ -140,13 +140,7 @@ ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
...
 endif::[]
 ifdef::openshift-online[]
-- You must register for an account with the https://openshift.com[{product-title} (Next Gen) Developer Preview.]
-- Your project must be populated with the
-xref:../dev_guide/templates.adoc#using-the-instantapp-templates[Instant App
-templates] and xref:../using_images/s2i_images/index.adoc#using-images-s2i-images-index[builder images] that
-conform to the current xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[set of
-considerations] for running in the {product-title} (Next Gen) Developer Preview
-offering.
+- You must register for an account with the https://manage.openshift.com.
```
Comment (sallyom) [inline getting_started/developers_cli.adoc]: @ahardin-rh @tiwillia  ok fixed!

**Review by tiwillia — APPROVED** [review]

**Comment by ahardin-rh** [comment]
Thanks! I am creating a new PR to update the {product-version} via the distro_map. This should be good to merge, however.

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**